### PR TITLE
Make syntax consistent. Reduce code. Remove if.

### PR DESCRIPTION
**Pull request by guyellis**

### CODE REVIEW

File: examples/flux-todomvc/js/actions/TodoActions.js [hunk 1]
```diff
@@ -42,17 +42,14 @@ var TodoActions = {
...
-        actionType: TodoConstants.TODO_UNDO_COMPLETE,
-        id: id
-      });
-    } else {
-      AppDispatcher.dispatch({
-        actionType: TodoConstants.TODO_COMPLETE,
-        id: id
-      });
-    }
+    var todoComplete = todo.complete ?
```
Comment (fisherwebdev) [inline examples/flux-todomvc/js/actions/TodoActions.js]: Naming here is confusing.  We can just name this variable `actionType`, or we can move both id and actionType down to be inline in the definition of the action object itself.

Comment (guyellis) [inline examples/flux-todomvc/js/actions/TodoActions.js]: Agreed. I changed the name to `actionType`.

**Comment by guyellis** [comment]
@fisherwebdev I just updated this PR with latest changes from master. Is it good to accept or do you need other changes?

**Comment by fisherwebdev** [comment]
Thanks!

**Pull request merged**